src/default-retrieve-file-handler.ts

```diff
@@ -1,23 +0,0 @@
-import * as fs from 'fs';
-import { transpileIfTypescript } from './transpile-if-ts';
-
-export function defaultRetrieveFileHandler(path) {
-  // Trim the path to make sure there is no extra whitespace.
-  path = path.trim();
-
-  // This was removed because it seems that we can't use cache while expecting correct results
-  // TODO: check correctness and performance with file caching 
-  // if (path in fileContentsCache) {
-  //   return fileContentsCache[path];
-  // }
-
-  var contents: string;
-  try {
-    contents = fs.readFileSync(path, 'utf8');
-    contents = transpileIfTypescript(path, contents);
```

Comment:
Yea, probably you are right. It seems that our tests were testing previous version instead of current. So, we were doing new stuff as blind cats and it was just luck that it works. So, I have to fix this... My bad, sorry...